piptools/scripts/compile.py

```diff
@@ -58,7 +58,7 @@ class PipCommand(pip.basecommand.Command):
               help=('Output file name. Required if more than one input file is given. '
                     'Will be derived from input file otherwise.'))
 @click.option('--allow-unsafe', is_flag=True, default=False,
-              help="Pin packages considered unsafe: pip, setuptools & distribute")
+              help="Pin packages considered unsafe: {}".format(list(UNSAFE_PACKAGES)))
```

Comment:
This is displaying as, `['distribute', 'pip', 'setuptools']` where previously it was displaying as `pip, setuptools & distribute`. Did you intend to include the full repr value of the list? Maybe do the following instead:

```python
', '.join(sorted(UNSAFE_PACKAGES)
```